code/game/turfs/open/floor.dm

```diff
@@ -45,7 +45,7 @@
 				stack_trace("[icon_state] (from [type]), which should be [burnt ? "NOT burnt, IS" : "burnt, IS NOT"]")
 				previous_errors[type] = TRUE
 
-	if(mapload && prob(33))
+	if(mapload && prob(7))
```

Comment:
I'd have to see the effects of this in-game to really judge them

Reply:
33 is 1/3, 7 is ~1/15, a really due change because of how disgusting the station can look. It's not just the floors, it's rusty walls, and broken flooring.

Reply:
This runs for every turf once- i. e. this is a chance per turf, not a chance for the whole event.

Reply:
> This runs for every turf once- i. e. this is a chance per turf, not a chance for the whole event.

That's wrong. The MakeDirty() function is... [Quite literally the dirtystation quirk.](https://github.com/Bubberstation/Bubberstation/blob/840ff9c2f978cad015ab0020778e513212749011/code/game/turfs/open/dirtystation.dm#L10)

Reply:
If you'd like, we could actually go directly to MakeDirty() and change each probability to make the station LESS dirty, if that's preferable.